src/controllerHost.ts

```diff
@@ -2,5 +2,6 @@ import {Whisper} from './whisper';
 import {PluginHost} from "./pluginHost";
 
 export interface ControllerHost extends PluginHost {
-  emitWhisper(whisper: Whisper): Promise<Error | void>;
+  emitWhisper(whisper: Whisper): Promise<Error | string>;
```

Comment:
```suggestion
  /**
   * @return - Promise resolving with the unique ID of the generated whisper.
   */
  emitWhisper(whisper: Whisper): Promise<Error | string>;
```
Let's add a description so we know what the string returned is?